tts/UI.def.lua

```diff
@@ -170,16 +197,38 @@ UI = {}
 ---@field children nil
 ---@field value nil | string
 
+---@shape tts__UITableLayoutElement : tts__UIElementBase
+
+---@shape tts__UIToggleGroupElement : tts__UIElementBase
+
+---@shape tts__UIToggleButtonElement_Attributes : tts__UIInputElementBase_Attributes
+---@shape tts__UIToggleButtonElement : tts__UIInputElementBase
+---@field attributes nil | tts__UIToggleButtonElement_Attributes
+---@field value nil | string
+
+---@shape tts__UIPanelElement_Attributes: tts__UIElementBase_Attributes
+---@field padding nil | tts__UIElement_Padding
+---@field color nil | tts__UIElement_Color
+
 ---@shape tts__UIPanelElement : tts__UILayoutElementBase
+---@field attributes nil | tts__UIPanelElement_Attributes
 ---@field tag "Panel"
 
+---@shape tts__UIOptionElement_Attributes: tts__UIElementBase_Attributes
```

Comment:
`---@shape Child : Parent` is preferred i.e. spaces surrounding `:`.

Reply:
Fixed several.